actions.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+generate-certificate:
```

Comment:
Since this action only works with self-signed certificates, this should be explicit. We could rename the action to `generate-self-signed-certificate` and specify it in the action description..

Reply:
Renamed the action name